app/models/manageiq/providers/azure_stack/cloud_manager/orchestration_stack/status.rb

```diff
@@ -1,4 +1,4 @@
-class ManageIQ::Providers::Azure::CloudManager::OrchestrationStack::Status < ::OrchestrationStack::Status
+class ManageIQ::Providers::AzureStack::CloudManager::OrchestrationStack::Status < ::OrchestrationStack::Status
```

Comment:
This little typo almost made me mad because orch stack was getting "Succeeded" status upon polling but `stack.status.completed?` was still reporting `false` :)